src/ibidem.rs

```diff
@@ -71,15 +73,30 @@ impl<P: syn::parse::Parse> std::ops::Deref for ParseWrapper<P> {
 
 /// While it is convenient to be able to "pass through" TokenStreams
 /// unperturbed, or to interpret TokenStreams via syn::parse::Parse.
+///
 /// While serde--wisely--does not permit this kind of unholy communion between
 /// Deserialize and Deserializer, we can skirt around this with the
 /// otherwise-unused deserialize_bytes/visit_bytes interfaces. Since there is
 /// no TokenStream that could reasonably be interpreted as bytes, we use this
-/// interface as a conduit to pass through a serialized form of a portion of
-/// the TokenStream from our Deserializer::deserialize_bytes() and deserialize
-/// it in WrapperVisitor::visit_bytes(). Yes, there is a serializer/deserializer
-/// pair buried *within* the broader deserialization. Gotta spend money to make
-/// money.
+/// interface to signal to the `serde_tokenstream` deserializer that we should
+/// be interpreting the TokenStream directly.
+///
+/// The signal works via a thread-local storage (TLS) side channel. When we
```

Comment:
```suggestion
/// This mechanism works via a thread-local storage (TLS) side channel. When we
```

Reply:
Done, thanks. (Good idea because the signal is part of the mechanism)